src/SWP/Bundle/ContentBundle/Resources/config/serializer/Model.Article.yml

```diff
@@ -1,5 +1,7 @@
 SWP\Bundle\ContentBundle\Model\Article:
     exclusion_policy: ALL
+    accessor_order: custom
+    custom_accessor_order: [id,title,body,slug,publishedAt,status,route,isPublishable,metadata,lead,code,sources,extra,slideshows,previousRelativeUrls,createdAt,updatedAt,authors,keywords,media,templateName,publishStartDate,publishEndDate,featureMedia,seoMetadata]
```

Comment:
why You want to maintain that order?

Reply:
because of behat tests requiring exact json content.

Reply:
probably would make more sense to improve these tests somehow?

Reply:
Yep, instead checking whole json we should check specific properties and their values.